tao_compiler/mlir/disc/disc_compiler.cc

```diff
@@ -320,9 +332,10 @@ LogicalResult LowerHLOToLLVM(ModuleOp m, const DISCLoweringOptions& options) {
   pm.addNestedPass<FuncOp>(
       disc_ral::createDiscUnhandledAtomicRMWConverterPass());
   pm.addNestedPass<FuncOp>(disc_ral::createDiscInputInlineFusionPass());
+  pm.addNestedPass<FuncOp>(arith::createArithmeticExpandOpsPass());
```

Comment:
That's because a lot of ops are migrating from std dialect to arithmetic dialect..... 
The expansion used to happen in StdExpandOps, now they should be in ArithmeticExpandOps. Like the expansion of MaxFOp.